lib/peep/storage/striped.ex

```diff
@@ -47,7 +49,7 @@ defmodule Peep.Storage.Striped do
 
   def insert_metric(tids, %Metrics.LastValue{} = metric, value, %{} = tags) do
     tid = get_tid(tids)
-    now = System.monotonic_time()
+    now = :erlang.monotonic_time()
```

Comment:
You can probably revert this one, it is inlined by the compiler anyway.

Reply:
```suggestion
    now = System.monotonic_time()
```